graphqlapi/src/main/java/io/stargate/graphql/schema/DmlSchemaBuilder.java

```diff
@@ -423,21 +443,44 @@ public GraphQLObjectType buildType(Table table) {
     GraphQLObjectType.Builder builder =
         GraphQLObjectType.newObject().name(nameMapping.getGraphqlName(table));
     for (Column columnMetadata : table.columns()) {
-      try {
-        GraphQLFieldDefinition.Builder fieldBuilder = buildOutputField(table, columnMetadata);
-        builder.field(fieldBuilder.build());
-      } catch (Exception e) {
-        log.error(String.format("Type for %s could not be created", columnMetadata.name()), e);
+      String graphqlName = nameMapping.getGraphqlName(table, columnMetadata);
+      if (graphqlName != null) {
+        try {
+          GraphQLFieldDefinition.Builder fieldBuilder =
+              new GraphQLFieldDefinition.Builder()
+                  .name(graphqlName)
+                  .type(fieldOutputTypes.get(columnMetadata.type()));
+          builder.field(fieldBuilder.build());
+        } catch (Exception e) {
+          warnings.add(
+              String.format(
+                  "Could not create output type for column %s.%s, skipping (%s)",
+                  columnMetadata.table(), columnMetadata.name(), e.getMessage()));
+        }
       }
     }
 
     return builder.build();
   }
 
-  private GraphQLFieldDefinition.Builder buildOutputField(Table table, Column columnMetadata) {
-    return new GraphQLFieldDefinition.Builder()
-        .name(nameMapping.getGraphqlName(table, columnMetadata))
-        .type(fieldOutputTypes.get(columnMetadata.type()));
+  private GraphQLFieldDefinition buildWarnings() {
+    StringBuilder description =
+        new StringBuilder(
+            "Warnings encountered during the translation of the backend schema into GraphQL.\n"
+                + "For convenience they are also listed below:");
```

Comment:
NIT: Can we change the message when there aren't any issues to avoid scaring the user, something like "No issues found when generating the GraphQL schema."